cypress/support/fragments/inventory/inventoryInstance.js

```diff
@@ -3,6 +3,7 @@ import QuickMarcEditorFragment from '../quickMarcEditor';
 import inventoryActions from './inventoryActions';
 import InventoryInstanceEdit from './InventoryInstanceEdit';
 import HoldingsRecordView from './holdingsRecordView';
+import inventoryViewSource from './inventoryViewSource';
```

Comment:
```
import inventoryActions 
import InventoryInstanceEdit 
import HoldingsRecordView 
import inventoryViewSource 
```
Please use first capital letter in imports

Reply:
fixed